docs/libcurl/curl_easy_unescape.3

```diff
@@ -39,7 +39,7 @@ that are URL encoded (%XX where XX is a two-digit hexadecimal number) are
 converted to their binary versions.
 
 If the \fBlength\fP argument is set to 0 (zero), \fIcurl_easy_unescape(3)\fP
-will use strlen() on the input \fIurl\fP string to find out the size.
+will use strlen() on the input \fBurl\fP string to find out the size.
```

Comment:
The argument in the synopsis is called **input** now, not url! (renamed recently)